apps/mailinglists/appconfig.py

```diff
@@ -4,3 +4,14 @@
 class MailinglistsConfig(AppConfig):
     name = "apps.mailinglists"
     verbose_name = "Mailinglists"
+
+    def ready(self):
+        super().ready()
+        # noinspection PyUnresolvedReferences
+        import apps.mailinglists.signals  # noqa: F401
+
+        from watson import search as watson
+
+        from apps.mailinglists.models import MailGroup
+
+        watson.register(MailGroup, fields=("public",))
```

Comment:
For actual results we should specify the text fields of the model. This boolean field doesn't really make sense in a text-search context